From acddc1b125f472270c1df83f54ff96e58ac08046 Mon Sep 17 00:00:00 2001
From: alexkar598 <25136265+alexkar598@users.noreply.github.com>
Date: Fri, 24 Dec 2021 16:47:06 -0500
Subject: Adds optimizations to the profile

---
 Cargo.toml | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/Cargo.toml b/Cargo.toml
index c19308e..5aed987 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -14,3 +14,8 @@ tokio = { version = "1", features = ["full"] }
 strum = "0.23"
 strum_macros = "0.23"
 phf = { version = "0.10", features = ["macros"] }
+
+[profile.release]
+lto = true
+codegen-units = 1
+panic = "abort"
\ No newline at end of file
-- 
2.34.1

