From 45cc025e6b612d9646c71bd1613f6138a6709c55 Mon Sep 17 00:00:00 2001
From: alexkar598 <25136265+alexkar598@users.noreply.github.com>
Date: Fri, 24 Dec 2021 17:29:22 -0500
Subject: [PATCH] Disable libudev requirement for serialport

---
 Cargo.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Cargo.toml b/Cargo.toml
index ea57897..c0dc461 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -8,7 +8,7 @@ edition = "2018"
 [dependencies]
 base64 = "0.13"
 nix = "0.23"
-serialport = "4.0"
+serialport = { version = "4.0", default-features = false }
 subprocess = "0.2"
 tokio = { version = "1", features = ["full"] }
 strum = "0.23"
-- 
2.34.1

