From 8e3f9b2851638c2b399797edc46bda05707007ff Mon Sep 17 00:00:00 2001
From: alexkar598 <25136265+alexkar598@users.noreply.github.com>
Date: Sat, 25 Dec 2021 01:27:57 -0500
Subject: [PATCH] Cringe fix for thing

---
 src/main.rs | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/main.rs b/src/main.rs
index 2185ae0..8c6b576 100644
--- a/src/main.rs
+++ b/src/main.rs
@@ -64,7 +64,7 @@ async fn main() {
                                     .expect("Error writing to serial");
                             }
                             Err(e) => {
-                                port.write_all(format!("{}ERR\0", encode(&e)).as_bytes())
+                                port.write_all(format!("{}\nERR\0", encode(&e)).as_bytes())
                                     .expect("Error writing to serial");
                             }
                         }
-- 
2.34.1

